asyncapi/json-schemas/sensors/telemetry/accelerometry/accelerometry.md

```diff
@@ -0,0 +1,35 @@
+### Accelerometry Message
+
+| Field         | Value                                                                                                     |
+|---------------|-----------------------------------------------------------------------------------------------------------|
+| Central Topic | ruter/{operatorId}/{vehicleId}/adt/v3/sensors/telemetry/01000008                                          |
+| Schema        | [ telemetry.json ](json-schemas/sensors/telemetry/telemetry.json)                                         |
+| Producer      | PTO                                                                                                       |
+| Consumer      | Ruter BO                                                                                                  |
+| Service Level | ✅ External API. Restrictions apply. Only backward compatible changes may happen within the major version. | 
+
+Describes the vehicle's acceleration in three dimensions.
+
+#### Data Specifications
+
+- **Frequency:** 6 times per minute (6/min)
```

Comment:
Burde vi kalle det "Message frequency", for å skille det fra bandwith, som også er en frekvens (måles i Hz)